html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+
+    <div>
+      <form action="index.html" method="get">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input id="email" type="text" name="email" maxlength="30">
+      </div>
+      </form>
+    </div>
+
+    <div>
+      <form action="index.html" method="get">
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <div>
+        <select id="why-is-coding-cool" name="reason">
+          <option value="select an option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      </form>
+    </div>
+
+    <div>
+      <form action="index.html" method="get">
+      <div>
+        <label for="favorite-language">Favorite Langauge (choose one)</label>
+      </div>
+      <div>
+        <input class="favorite-language" type="radio" name="language" value="html"><span>HTML</span>
```

Comment:
The text you have in `span` elements here is what should be in a `label` element. Make sure that they all have a `for` attribute that correlates to the `id` on the input it is paired with.